a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -1099,3 +1099,43 @@ def execute(self, subnet, use_device_flavor, members):
                               subnet.id, str(e))
                 raise e
         return 0
+
+
+class CountBatchMembersWithIPPortProtocol(BaseDatabaseTask):
+    def execute(self, members, pool):
+        try:
+            mem_count_ip_dict = {}
+            mem_count_ip_port_protocol_dict = {}
+            for member in members:
+                cnt_ip = self.member_repo.get_member_count_by_ip_address(
```

Comment:
maybe we can skip the database query if mem_count_ip_dict[member.ip_address] already exist